lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -312,10 +315,25 @@ def get(self, request, *args, **kwargs):
                     )
                 ] if 'children' in chapter_data else []
 
+        # Get has_scheduled_content data
+        collected_block_structure = get_block_structure_manager(course_key).get_collected()
+        transformers = BlockStructureTransformers()
+        transformers += [start_date.StartDateTransformer(), ContentTypeGateTransformer()]
```

Comment:
You don't actually need ContentTypeGateTransformer here. The progress tab adds it for other reasons unrelated to `has_scheduled_content`. (And should probably have a comment to that effect - my fault.